vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -152,7 +170,7 @@ private void checkIfThereAreSourceForEveryStudy(List<String> studyIds,
         for (VariantSource source : sources) {
             // create a map from original to "conflict free" sample name (prefixing with study id)
             Map<String, String> fileSampleNamesMapping = new HashMap<>();
-            source.getSamples().stream()
+            source.getSamplesPosition().keySet().stream()
```

Comment:
I think there were some tests for this, do the test pass with the new code?

Reply:
Yes